[2024-04-30T16:54:14.146+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: etl_reddit_pipeline.reddit_image_extraction manual__2024-04-30T16:46:29.174545+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
